application/models/SurveyDynamic.php

```diff
@@ -29,7 +29,7 @@ class SurveyDynamic extends LSActiveRecord
     /** @var int $sid */
     protected static $sid = 0;
 
-    /** @var Survey $survey */
+    /** @var array $survey */
```

Comment:
Must update

https://github.com/LimeSurvey/LimeSurvey/blob/e16a96b5276a2508eb5920609dcc32e27bd6b9a8/application/models/SurveyDynamic.php#L170
https://github.com/LimeSurvey/LimeSurvey/blob/e16a96b5276a2508eb5920609dcc32e27bd6b9a8/application/models/SurveyDynamic.php#L187
https://github.com/LimeSurvey/LimeSurvey/blob/e16a96b5276a2508eb5920609dcc32e27bd6b9a8/application/models/SurveyDynamic.php#L203
https://github.com/LimeSurvey/LimeSurvey/blob/e16a96b5276a2508eb5920609dcc32e27bd6b9a8/application/models/SurveyDynamic.php#L897
https://github.com/LimeSurvey/LimeSurvey/blob/e16a96b5276a2508eb5920609dcc32e27bd6b9a8/application/models/SurveyDynamic.php#L1041


maybe move this part to another PR ?

Reply:
See above. Used a getter now. Will return the same object now.